src/server/scripts/Kalimdor/zone_desolace.cpp

```diff
@@ -25,8 +25,6 @@ EndScriptData */
 /* ContentData
 npc_aged_dying_ancient_kodo
 go_iruxos
```

Comment:
```suggestion
```
This line will effectively also be redundant when its script is moved to SAI.

Reply:
that's why i removed it

Reply:
Yes, but I suggest removing this comment line too.

Reply:
Thanks. You can now mark this conversation as resolved, to minimize it. :)